### PR TITLE
Change in a lot of files

### PR DESCRIPTION
**Pull request by AndersBensen**
Kasper and Anders changed the interaction with the cook so the window, button and text changed depending on the situation.  Fixed a bug so you now need to be in the same room as the cook to talk with him. FInally we also checked every file and wrote missing javadocs, removed spaces and unnecessary lines.

### CODE REVIEW

File: src/gui/GameController.java [hunk 1]
```diff
@@ -99,10 +106,11 @@ public void updateGameState() {
     }
     
     /**
-     * Method to check if Cook and Player are in the same room.
+     * Method to check if Cook and Player are in the same room and if the player has a steak with or without ketchup.
      */
     public void inSameRoom() {
-        if (game.getPlayer().getCurrentRoom() == game.getCook().getCurrentRoom() && !game.getPlayer().hasItem("Steak")) {
+        if (game.getPlayer().getCurrentRoom() == game.getCook().getCurrentRoom() && !game.getPlayer().hasItem("Steak")
+                && !game.getPlayer().hasItem("Steak-with-ketchup")) {
```
Comment (quartercastle) [inline src/gui/GameController.java]: Hvis i newliner if sætningen som herunder er det nemmer at overskue.
```java
if (
    game.getPlayer().getCurrentRoom() == game.getCook().getCurrentRoom() &&
    !game.getPlayer().hasItem("Steak") && 
    !game.getPlayer().hasItem("Steak-with-ketchup")
) {
     gui.getStages().get("CookInteraction").show();
}
```

File: src/maga/util/Console.java [hunk 2]
```diff
@@ -1,5 +1,6 @@
 package maga.util;
 
+// should probably be deleted
```
Comment (quartercastle) [inline src/maga/util/Console.java]: Må gerne slettes.

**Review by quartercastle — CHANGES_REQUESTED** [review]

File: src/gui/GameController.java (hunk 1 above)
Comment (AndersBensen) [reply src/gui/GameController.java]: fixed in: 5bfd32fc4f668a057efc1cbd6dcd8cc34d3e5f0a

File: src/maga/util/Console.java (hunk 2 above)
Comment (AndersBensen) [reply src/maga/util/Console.java]: fixed in: 5bfd32fc4f668a057efc1cbd6dcd8cc34d3e5f0a

**Review by quartercastle — APPROVED** [review]

**Pull request merged**